src/DesktopClient/PresenceLight/PresenceLight.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
This is required for Razor to copy the various static files around (HTML, JS, CSS, etc.).